legal-api/report-templates/certificateOfNameChange.html

```diff
@@ -0,0 +1,68 @@
+[[macros.html]]
```

Comment:
Looks good. Can you please attach a sample PDF generated using this, to this PR or the ticket, to demonstrate that it looks OK? Thanks!

Reply:
[certificateOfNameChange.pdf](https://github.com/bcgov/lear/files/6012082/certificateOfNameChange.pdf)